### ISSUE TITLE
Enforce modifier order

### ISSUE THREAD

**Issue by jhaber** [jhipster/prettier-java#291]
I think it would be in-scope for prettier-java to rewrite class, field, and method modifiers to match the order specified in the Java Language Specification. Here's the order that checkstyle uses:
https://checkstyle.sourceforge.io/config_modifier.html#ModifierOrder

I was testing with this code:
```java
public class Test {

  static public void main(String... args) {}
}
```
So the desired output would be:
```java
public class Test {

  public static void main(String... args) {}
}
```

**Comment by jhaber** [comment]
If you agree it's appropriate for prettier-java to handle this, I can attempt to send a PR in the next few weeks

**Comment by clement26695** [comment]
Good idea ! Please go ahead: if you need any help, let me know !

**Comment by clementdessoude** [comment]
Closed with #302
